docs/source/staking/running_a_worker.rst

```diff
@@ -0,0 +1,426 @@
+.. _ursula-config-guide:
+
+================
+Running a Worker
+================
+
+NuCypher staking operations are divided into two roles "Staker" and "Worker" - This Guide is for Workers.
+
+Overview
+----------
+
+Worker's role in the network
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Worker is the bonded delegate of a Staker and an active network node.  Each staking account
+or "Staker" is bonded to exactly one Worker.  Workers must remain online to provide uninterrupted
+re-encryption services to network users on-demand and perform periodic automated transactions to
+signal continued commitment to availability.
+
+
+Workers nodes have three core components
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Ethereum software wallet (keystore)
+* Local or hosted ethereum provider
+* Worker node; Local or cloud server
+
+
+Minimum system requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* 20GB storage
+* 2GB RAM
+* x86 architecture
+* Static IP address
+* Exposed TCP port 9151
+
+Workers can be run on cloud infrastructure – for example,
+`Digital Ocean 4GB Basic Droplet <https://www.digitalocean.com/pricing/>`_ satisfies requirements listed above.
+
+
+Best Practices
+^^^^^^^^^^^^^^
+
+**The state of worker diligence**
+
+Workers can demonstrate a vested interest in the success of the network by adhering to
+three core areas of responsibility (in order of importance):
+
+#1 Keystore Diligence
+
+Requires that the custodian keep track of a secret seed which can be used to generate the entire keystore.
+
+    - Keep an offline backup up mnemonic seed phrases.
+    - Use a password manager to generate a strong password when one is required.
+
+#2 Datastore Diligence
+
+Requires that material observed during the runtime be stored.
+A running worker stores peer metadata, re-encryption key fragments ("Kfrags"), and "treasure maps".
+
+Loss of stored re-encryption key fragments will indicate slashing on the bonded stake.
+If a worker node has already agreed to enforce a policy, then loses a Kfrag, network users
+can issue a challenge which is verified onchain by the Adjudicator contract.
+
+As a civic matter, datastore diligence is important for Ursula for several reasons
+Including storing node validity status (and thus refraining from pestering nodes
```

Comment:
meant to be bullets?

Reply:
Trying to make paragraphs, although they can use some love from @jMyles :-)
Some of these best practices are subject to changing with #2530.